modules/apps/sync/sync-engine/src/main/java/com/liferay/sync/engine/document/library/handler/GetSyncDLObjectUpdateHandler.java

```diff
@@ -309,6 +309,62 @@ protected void addFile(SyncFile syncFile, String filePathName)
 		}
 	}
 
```

Comment:
checkUnsynced() follows the same unsynced logic in the deleted moveFile() method.